configs/default/AFNG-ALIENFLIGHTNGF7_ELRS.config

```diff
@@ -13,6 +13,8 @@
 #define USE_SDCARD
 #define USE_RX_EXPRESSLRS
 #define USE_RX_EXPRESSLRS_TELEMETRY
+#define RX_SPI_DEFAULT_PROTOCOL          RX_SPI_EXPRESSLRS
+#define RX_EXPRESSLRS_TIMER_INSTANCE     TIM5
```

Comment:
These already are handled in common_post.h - but might be needed for a planned cleanup I'm not aware off :)

Reply:
For config.h cleanup. I've removed from common post.